src/controllers/package.js

```diff
@@ -8,12 +13,47 @@ export const updatePackagesController = (id, price) => {
     return updatePackagesModel(id, price)
 }
 
+export const insertPackageController = async (id, packageDetails) => {
+    const availiable = await getAvailiableSession()
+    if (availiable > 0) {
+        console.log(packageDetails, availiable)
+        await insertPackageModel(id, packageDetails, availiable)
+        return true;
+    }
+    return false;
+}
+
+export const displayRequestPackageController = async () => {
+    return await getRequestpackage()
```

Comment:
Since you are not using the value coming from getRequestPackage in this controller function, you can choose to return the Promise to the handler and leave it to resolve it.

In short, return getRequestPackage() is fine. Also correct the typo. getRequestpackage getRequestPackage